### PR TITLE
Issue27 - test_max_limit - keep it to 50K max by setting index.max_result_window returned from elasticsearch

### PR DESCRIPTION
**Pull request by maggienj**

### CODE REVIEW

File: resources/config/elasticsearch.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ index.number_of_shards: 1
 index.number_of_replicas: 1
 index.translog.interval: 60s
 index.translog.flush_threshold_size: 1gb
+index.max_result_window=50000
```
Comment (klahnakoski) [inline resources/config/elasticsearch.yml]: You should not need this line

Comment (maggienj) [inline resources/config/elasticsearch.yml]: Agreed. Please ignore this pull request. 
Will send a new pull request from issue27a, where in, only the required files will be touched.

Comment (klahnakoski) [inline resources/config/elasticsearch.yml]: as per https://github.com/maggienj/ActiveData/issues/40 this line should be removed

File: resources/schema/basic_schema.json.template [hunk 2]
```diff
@@ -14,6 +15,7 @@
 							"type": "double",
 							"store": true
 						},
+
```
Comment (klahnakoski) [inline resources/schema/basic_schema.json.template]: remove this blank line.

**Pull request closed without merging**